eternalcombat-plugin/src/main/java/com/eternalcode/combat/crystalpvp/CrystalPvpConstants.java

```diff
@@ -0,0 +1,95 @@
+package com.eternalcode.combat.crystalpvp;
+
+import com.eternalcode.combat.config.implementation.PluginConfig;
+import com.eternalcode.combat.fight.FightManager;
+import com.eternalcode.combat.fight.event.CauseOfTag;
+import com.eternalcode.combat.util.ReflectUtil;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.bukkit.Material;
+import org.bukkit.block.BlockState;
+import org.bukkit.entity.EnderCrystal;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDamageByBlockEvent;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.metadata.MetadataValue;
+
+public class CrystalPvpConstants {
```

Comment:
I don't know if this class is named right @Rollczi?

Reply:
If `Pvp` has only the first letter uppercased, then the method below should be renamed to getDamager`Uuid`FromEndCrystal